surveys/2024-annual-survey/translations/de.md

```diff
@@ -183,8 +183,8 @@ Type: select one
 - 1.79
 - 1.78
 - 1.77
-- 1.76
-- 1.75
+- 1,76
+- 1,75
```

Comment:
```suggestion
- 1.76
- 1.75
```

(version numbers are separator-sensitive, probably can't be localized)

Reply:
All commas in rustc version numbers have been fixed in https://github.com/rust-lang/surveys/commit/945c1edb8a2c6d55e8d619dba6c7613ac94bd258

Reply:
Oops. Typo, sorry.

Reply:
I fixed the two commas in SurveyHero and reverted them from this PR.